### PR TITLE
bump lance version for vectordb, fix minor bugs in lancedb remote client

### PR DESCRIPTION
**Pull request by chebbyChefNEQ**

### CODE REVIEW

File: python/lancedb/remote/client.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ async def get(self, uri: str, params: Union[Dict[str, Any], BaseModel] = None):
     async def post(
         self,
         uri: str,
-        data: Union[Dict[str, Any], BaseModel, bytes],
+        data: Union[Dict[str, Any], BaseModel, bytes, None] = None,
```
Comment (eddyxu) [inline python/lancedb/remote/client.py:108]: Why do you need `Non` in the Union?

Can we just use `Optional[Union[...]]`?

File: python/lancedb/remote/db.py [hunk 2]
```diff
@@ -103,7 +103,7 @@ def create_table(
 
         self._loop.run_until_complete(
             self._client.post(
-                f"/v1/table/{name}/create",
+                f"/v1/table/{name}/create/",
```
Comment (eddyxu) [inline python/lancedb/remote/db.py:106]: Would there be a  integration tests ?

**Review by eddyxu — APPROVED** [review]
Minor comments.

File: python/lancedb/remote/db.py (hunk 2 above)
Comment (chebbyChefNEQ) [reply python/lancedb/remote/db.py:106]: yes, I caught in integration test.

**Pull request merged**